cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -433,3 +433,18 @@ endif()
 # complain about unused CMake variable.
 if (CMAKE_CROSSCOMPILING_EMULATOR)
 endif()
+
+# TODO: CMake appends <sysroot>/usr/include to implicit includes; switching to use usr/include will make this redundant.
+set(_em_sysroot_include "${EMSCRIPTEN_SYSROOT}/include")
+if("${CMAKE_VERSION}" VERSION_LESS "3.20")
+  file(TO_CMAKE_PATH "${_em_sysroot_include}" _em_sysroot_include)
+else()
+  cmake_path(CONVERT "${_em_sysroot_include}" TO_CMAKE_PATH_LIST _em_sysroot_include)
+endif()
```

Comment:
Why is the above if/else block needed?

Reply:
I think I assumed that "superseded" in the documentation implied "deprecated" (in context of file vs cmake_path), but on second glance that might not be true. I'll simplify it.